src/xc_integrator/local_work_driver/device/cuda/kernels/uvvars.cu

```diff
@@ -165,14 +165,15 @@ __global__ void eval_uvars_lda_uks_kernel( size_t        ntasks,
 
 
   if( tid_y < npts ) {
-    den_pos_eval_device[ tid_y ] += den_neg_eval_device[ tid_y ];
-    den_neg_eval_device[ tid_y ] *= -2.0;
-    den_neg_eval_device[ tid_y ] += den_pos_eval_device[ tid_y ];
+		const auto ps = den_pos_eval_device[ tid_y ];
```

Comment:
Per the other comment - this is definitely a tabs/spaces issue. We use spaces :).